keras/layers/preprocessing/center_crop.py

```diff
@@ -111,6 +111,13 @@ def call(self, inputs):
 
     def compute_output_shape(self, input_shape):
         input_shape = list(input_shape)
+        if isinstance(input_shape[0], (list, tuple)) or len(
```

Comment:
That should be `not` & `and`

Reply:
I am not getting exactly. First condition I am checking whether `input_shape` is nested. If nested that should raise error. Second condition is checking input should be either 3D or 4D. If either of these condition fails it should raise error.

Reply:
Got it, I misread that.